direct-message-key.js

```diff
@@ -29,40 +30,14 @@ function directMessageKey (my_dh_secret, my_dh_public, my_feed, your_dh_public,
   const info = slp.encode([info_context, ...info_keys])
 
   return hkdf(input_keying_material, length, { salt, info, hash })
-}
-
-function SHA256 (input) {
-  const hash = crypto.createHash('sha256')
-
-  hash.update(input)
-  return hash.digest()
-}
-
-directMessageKey.easy = EasyDirectMessageKey
 
-function EasyDirectMessageKey (ssbKeys) {
-  const my = {
-    dh: new DHKeys(ssbKeys, { fromEd25519: true }).toBFE(),
-    feedId: bfe.encode(ssbKeys.id)
-  }
-
-  return function EasyDirectMessageKey (feedId) {
-    const dotIndex = feedId.indexOf('.')
-    const keys = {
-      public: feedId.substring(1, dotIndex > 0 ? dotIndex : feedId.length)
-      // prunes the @ + suffix
-    }
-
-    const your = {
-      dh: new DHKeys(keys, { fromEd25519: true }).toBFE(),
-      feedId: bfe.encode(feedId)
-    }
-
-    return directMessageKey(
-      my.dh.secret, my.dh.public, my.feedId,
-      your.dh.public, your.feedId
-    )
+  // TODO
```

Comment:
@arj03 are you down for this change?

I want to return the SCHEME here too, so that developers don't have to think about it.
We have some awkwardness around our language where what we put in a `key_slot`  of the  `envelope` function is not just a key, but a `{ key, scheme }` pair...  

anyway this would be a breaking change for this API, but I think it would be a net positive

Reply:
Yes, I think that would be a good idea.

Reply:
Shouldn't the returned scheme be a string, I see you did them all as buffers?

Reply:
yeah turns out it's safe to pass the schema as buffer because it immediately gets converted to a buffer if it isn't one already